automates/text_reading/src/main/resources/org/clulab/aske_automates/grammars/descriptions.yml

```diff
@@ -311,6 +311,25 @@ rules:
     pattern: |
       [word=/a|an|the/] (?<description> [tag=/JJ|VBN/] [tag=/^N/] ) @variable:Identifier (?![tag=/^N/])
 
+  - name: at_rate
```

Comment:
If possible, could you add examples to the newly added rules? (at_rate, at_rate_with_verb, identifier_def_ideal_table) That way it would be clearer for me what these rules are for.

Reply:
Added examples for the at_rate rules; getting rid of identifier_def_ideal_table---that was to specific for one of the papers. Good catch!

Reply:
Thanks for adding the examples! But I'm still not getting what we're aiming to extract as a definition here. What is extracted as the definition for _ls_ in this sentence?: Susceptible individuals acquire infection with strain 1 at a rate Ic.

Reply:
Could you explain a little more about the context of that example sentence and the rule itself? Maybe in our meeting tomorrow?

Reply:
Basically, trying to extract as much information as possible to allow good alignment under the conditions of no real description, so ideally, I'd get "Susceptible individuals acquire infection with strain 1 at a rate" as description. I think because of expansion or something like that I grab the identifier itself as part of the description, but that's a whole other issue that needs to be fixed for some other cases too and was a necessary evil.

Reply:
Oh, I see. That makes sense now. Thank you for the explanation! I was confused because I couldn't find any definition there. But I guess this rule is not for extracting real definitions but for extracting information needed for improving alignment.